cmd/policy.go

```diff
@@ -53,32 +52,10 @@ var (
 	}
 
 	policyDownloadCmd = &cobra.Command{
-		Use:   "download GITHUB_REPO",
-		Short: policyDownloadHelpMsg,
-		Long:  policyDownloadHelpMsg,
-		Example: `
-  # Download official policy
-  cloudquery policy download aws
-  
-  # The following will be the same as above
-  # Official policies are hosted here: https://github.com/cloudquery-policies
-  cloudquery policy download aws//cis-1.2.0
-	
-  # Download community policy
-  cloudquery policy download github.com/COMMUNITY_GITHUB_ORG/aws
-
-  # See https://hub.cloudquery.io for additional policies.`,
-		Args: cobra.ExactArgs(1),
+		Use:        "download",
+		Deprecated: "Use 'policy run' command directly instead. If you need to download a policy to your machine, you can use 'git clone [https://github.com/cloudquery-policies/aws.git]'. See https://docs.cloudquery.io/docs/cli/policy/sources. ",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			c, err := console.CreateClient(cmd.Context(), getConfigFile(), true, nil, instanceId)
-			if err != nil {
-				return err
-			}
-			diags := c.DownloadPolicy(cmd.Context(), args)
-			if diags.HasErrors() {
-				return fmt.Errorf("policy download has one or more errors, check logs")
-			}
-			return nil
+			return fmt.Errorf("'policy download' command has been deprecated. Use the 'policy run' command directly instead")
```

Comment:
This should have a link to our documentation on how users should be downloading policies